main.go

```diff
@@ -369,12 +370,18 @@ func serve(conf config.Config) error {
 		negroni.Wrap(http.HandlerFunc(logoutHandler)),
 	)).Methods("GET")
 
+	allowCORS := handlers.CORS(
+		handlers.AllowedOrigins([]string{"*"}),
+		handlers.AllowedMethods([]string{"GET", "POST", "OPTIONS"}),
+		handlers.AllowedHeaders([]string{"Content-Type", "X-Grpc-Web", "content-type", "x-grpc-web"}),
+	)
+
 	// All websockets requests are POSTs to some {method} on this route.
 	p.Handle("/web.Proxy/{method}", negroni.New(
 		setConf(conf),
 		negroni.HandlerFunc(withLog),
 		negroni.HandlerFunc(authHandler),
-		negroni.Wrap(websocketsProxy(wsproxy)),
+		negroni.Wrap(allowCORS(websocketsProxy(wsproxy))),
 	)).Methods("POST")
```

Comment:
All grpc requests go to `/web.Proxy/{methodName}`. Do we need to explicitly add OPTIONS handling somewhere in here?